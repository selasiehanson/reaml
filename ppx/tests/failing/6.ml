let _foo =
 fun [@reaml.component "Foo"] () ->
  let _ =
   fun () ->
    let[@reaml] () = () in
    ()
  in
  ()
