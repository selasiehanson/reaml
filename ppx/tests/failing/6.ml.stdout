[%%ocaml.error ;;"[@reaml] occurs in an inappropriate place"
  ;;""]
